packages/fast-tooling-react/src/css-property-editor/property-editor-row.props.ts

```diff
@@ -0,0 +1,62 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+export interface CSSPropertyEditorRowClassNameContract {
+    cssPropertyEditorRow?: string;
+    cssPropertyEditorRow_input?: string;
+    cssPropertyEditorRow_inputKey?: string;
+    cssPropertyEditorRow_inputValue?: string;
+}
+
+export interface CSSPropertyEditorRowUnhandledProps
+    extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface CSSPropertyEditorRowHandledProps
+    extends ManagedClasses<CSSPropertyEditorRowClassNameContract> {
+    /**
+     * The key associated with the value being edited
+     */
+    rowIndex: number;
+
+    /**
+     * The key associated with the value being edited
+     */
+    cssKey: string;
+
+    /**
+     * The value for the property being edited
+     */
+    value: string;
+
+    /**
+     * Called when the value changes
+     */
+    onValueChange: (newValue: string, rowKey: string, rowIndex: number) => void;
+
+    /**
+     * Called when the key value changes
+     */
+    onKeyChange: (newkey: string, oldKey: string, rowIndex: number) => void;
+
+    /**
+     * Key input has lost focus
+     */
+    onCommitKeyEdit: (rowKey: string, rowIndex: number) => void;
```

Comment:
can you update the arguments on these callbacks to `rowKey` -> `key` and `rowIndex` -> `id`

Reply:
rowIndex is actually an index, not an id, we could use "index" but personally I find "rowIndex" more descriptive.   For consistency we should probably use the same name for rowKey that we use for cssKey (in your previous comment) but that can't be simply key.